integration-cli/docker_cli_network_unix_test.go

```diff
@@ -67,14 +66,10 @@ func (s *DockerNetworkSuite) SetUpSuite(c *check.C) {
 		fmt.Fprintf(w, "null")
 	})
 
-	if err := os.MkdirAll("/etc/docker/plugins", 0755); err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(os.MkdirAll("/etc/docker/plugins", 0755), check.IsNil)
```

Comment:
@HuKeping in these case I tend to do something like 
```go
err := os.MkdirAll("/etc/docker/plugins", 0755)
c.Assert(err, checker.IsNil)
```

Just so that everything is not packed :stuck_out_tongue_closed_eyes: